metadata_mapper/mappers/mapper.py

```diff
@@ -350,7 +323,7 @@ def shred(self, prop, delim=";"):
 
         return self
 
-    def copy_prop(self, prop, to_prop, skip_if_exists=[False]):
+    def copy_prop(self, prop, to_prop, no_overwrite=None, skip_if_exists=[False]):
```

Comment:
did you find the one case of `no_overwrite` as a parameter??? which collection was that?

Reply:
I didn't make a note of it, a decision I now regret. 😞

Reply:
It's here: https://registry.cdlib.org/api/v1/rikolticollection/3433/?format=json